src/types/Schedule.ts

```diff
@@ -1,5 +1,5 @@
 export interface Schedule {
-    dateOfWeek: string;
-    start: string;
-    end: string;
+    dateOfWeek: number;
```

Comment:
Renaming this to `dayOfWeek` makes a bit more sense (you'll also have to update it elsewhere but a replace all should work).

Can we also add a comment saying how numbers refer to days of week (ex: is Monday = 0?)